### PR TITLE
rdbc-579

### PR DESCRIPTION
**Pull request by alxsabo**
https://issues.hibernatingrhinos.com/issue/RDBC-579/PHP-client-compare-exchange

### CODE REVIEW

File: tests/Test/Client/_CrudTest/Entities/MemberArray.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace tests\RavenDB\Test\Client\Crud\Entities;
+namespace tests\RavenDB\Test\Client\_CrudTest\Entities;
```
Comment (ml054) [inline tests/Test/Client/_CrudTest/Entities/MemberArray.php:3]: why we had to change this namespace?

File: tests/Test/Client/_QueryTest/Entity/Article.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace tests\RavenDB\Test\Client\Query\Entity;
+namespace tests\RavenDB\Test\Client\_QueryTest\Entity;
```
Comment (ml054) [inline tests/Test/Client/_QueryTest/Entity/Article.php:3]: same here

File: composer.json [hunk 3]
```diff
@@ -24,6 +24,7 @@
         "doctrine/annotations": "^1.13",
         "doctrine/inflector": "^2.0",
         "ramsey/uuid": "^4.2",
+        "spatie/async": "^1.5",
```
Comment (ml054) [inline composer.json]: do we still need that?

Comment (alxsabo) [reply composer.json]: removed

File: tests/Test/Client/_CrudTest/Entities/MemberArray.php (hunk 1 above)
Comment (alxsabo) [reply tests/Test/Client/_CrudTest/Entities/MemberArray.php:3]: explained in direct call

**Pull request merged**